README.md

```diff
@@ -35,82 +35,146 @@ Connect joy via nano USB receiver to NUC and make sure it is in **DirectInput Mo
-```
+    ```bash
+    ssh husarion@10.15.20.3
+    ```
 
-default password is `husarion`
+    The default password is `husarion`
 
 2. Clone this repo
 
-```bash
-cd ~/husarion_ws/src/
-git clone https://github.com/husarion/kortex_joystick_demo.git
-```
+    ```bash
+    cd ~/husarion_ws/src/
+    git clone https://github.com/husarion/kortex_joystick_demo.git
+    ```
 
-3. Setup gamepad according to [Setup joy](#setup-joy)
+3. Setup a gamepad according to [Setup joy](#setup-joy)
 
 4. Setup virtual desktop
 
-```bash
-cd kortex_joystick_demo/
-. ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
-```
+    ```bash
+    cd kortex_joystick_demo/
+    . ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
+    ```
 
 5. Launch demo
 
-```bash
+    ```bash
+    ```
 
-docker compose -f compose.panther-kinova.yaml up
-```
+<!-- More info needed here TBD -->
 
-After successful launch open Web browser and go to [10.15.20.3:8080](http://10.15.20.3:8080/vnc_auto.html). Enter password (default `husarion`) and you should see RViz and Panther with manipulator.
+After a successful launch open the Web browser and go to [10.15.20.3:8080](http://10.15.20.3:8080/vnc_auto.html). Enter a password (default `husarion`) and you should see RViz and Panther with the manipulator.
 
-## Controlling Kinova manipulator
+<!-- --- -->
 
-By default, after a successful launch manipulator will be controlled by the gamepad. To start driving and disable the manipulator press **DISABLE MOVEMENT** (**LB** button). To switch back to using the manipulator press **ENABLE MOVEMENT** (**A** button).
+<!-- ## Docker image
 
-### Manipulator button mapping
+[![Build/Publish Docker Image](https://github.com/husarion/kortex_joystick_demo/actions/workflows/build-docker-image.yaml/badge.svg)](https://github.com/husarion/kortex_joystick_demo/actions/workflows/build-docker-image.yaml)
 
-|  Button  |      Function      |
-|:--------:|:------------------:|
-|   `LB`   |   disable movement |
-|   `LT`   |    close gripper   |
-|   `RT`   |    open gripper    |
-|   `B`    |   emergency stop   |
-|   `A`    |   enable movement  |
-|   `X`    |    linear mode     |
-|   `Y`    |    angular mode    |
-|   `UP CROSS KEY`    |    increase speed    |
-|   `DOWN CROSS KEY`  |    decrease speed    |
+| ROS distro | Supported architectures      |
+| ---------- | ---------------------------- |
+| `noetic`   | `linux/amd64`, `linux/arm64` |
 
-![buttons](docs/gamepad_buttons.png)
+Available on [Docker Hub](https://hub.docker.com/r/husarion/kortex_joystick_demo/tags)
 
-To move manipulator use sticks.
-By default, cartesian linear/angular movement in `X` and `Y` are held by left stick and linear/angular `Z` movement is controlled with right stick.
-To switch between **linear** and **angular** mode use **X** and **Y** buttons. 
+### Demo
 
-## Controlling robot
+#### Panther robot with Kinova manipulator installed on the top
 
-### Robot button mapping
+1. Clone this repo on your ROSbot:
```

Comment:
Probably instead of ROSbot should be Panther.

Reply:
☑️